corehq/ex-submodules/casexml/apps/phone/management/commands/sync_log_debugger.py

```diff
@@ -51,3 +60,13 @@ def handle(self, *args, **options):
                             log_names[j],
                             ', '.join(case_diff)
                         )
+
+        if options['debugger']:
+            union_of_ids = set().union(*[set(log.get_footprint_of_cases_on_phone()) for log in logs])
```

Comment:
(these are just useful to have in local context when you drop into a debugger on the next lines)